packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -1452,6 +1436,46 @@ describe.each([
         { status: 404 }
       )
     })
+
+    // MSSQL needs a setting called IDENTITY_INSERT to be set to ON to allow writing
+    // to identity columns. This is not something Budibase does currently.
+    providerType !== DatabaseName.SQL_SERVER &&
+      it("should handle filtering by composite primary keys", async () => {
+        const tableRequest = saveTableRequest({
+          primary: ["number", "string"],
```

Comment:
The last time I was looking into this, for external data sources, only the first element in this array gets used. Are you sure that this test is actually creating a composite primary key?

Reply:
Not 100% sure if in the database it gets created as identity, but for the logic of these export tests we shoudn't really care. We want to treat both columns as the primary key, and we want to get sure that the select queries work as expected (ignoring if they are or are not primary keys on the external dbs).

Even so, we could double-check it, but I think it gets actually set on the DB. For this reason, I needed to disable SQL Server, as we were getting actual DB exceptions

Reply:
Just to clarify, without this PR code changes, this test would fail